[rpc][flaky-tests] fix for test_handle_send_exceptions and
backward_node_failure by adding more error regexes

rohan-varma opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #32659 apply linter to rpc test files
* **#32656 [rpc][flaky-tests] fix for test_handle_send_exceptions and
backward_node_failure by adding more error regexes**
backward_node_failure by adding more error regexes**
backward_node_failure by adding more error regexes**
backward_node_failure by adding more error regexes**

backward_node_failure by adding more error regexes

Fixes these flaky tests.

Differential Revision: [D19584453](https://our.internmc.facebook.com/intern/diff/D19584453/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D19584453/)!

rohan-varma committed:
[rpc][flaky-tests] fix for test_handle_send_exceptions and
backward_node_failure by adding more error regexes

Fixes these flaky tests.

Differential Revision: [D19584453](https://our.internmc.facebook.com/intern/diff/D19584453/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D19584453/)!

[ghstack-poisoned]

pritamdamania87 commented on torch/testing/_internal/distributed/rpc/dist_autograd_test.py:
nit: just use `get_shutdown_error_regex` inline instead of having a separate `error_regex` variable.

pritamdamania87 reviewed (approved):
Looks good to me! Please fix link issues before landing.

rohan-varma committed:
Update on "[rpc][flaky-tests] fix for test_handle_send_exceptions and
backward_node_failure by adding more error regexes"

backward_node_failure by adding more error regexes**
backward_node_failure by adding more error regexes**

backward_node_failure by adding more error regexes

Fixes these flaky tests.

Differential Revision: [D19584453](https://our.internmc.facebook.com/intern/diff/D19584453/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D19584453/)!

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 9ad0f1b4:


**None of the build failures appear to be your fault**.

* **1/1** [broken upstream](https://dr.pytorch.org/master-timeline.html?count=1&sha1=5b321a098594693eca8408169f475cb334856655&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35) at merge base 5b321a09 since Jan 27
  <details>
  <summary>
  Please rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  Since your merge base is older than `viable/strict`, run these commands:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=9ad0f1b4cd0a4e705c434b747f072434b1931086).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably **caused by upstream breakages**:
* [pytorch_macos_10_13_py3_test](https://dr.pytorch.org/build-details.html?build_id=91469987) from Jan 27 until Jan 27



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

rohan-varma committed:
Update on "[rpc][flaky-tests] fix for test_handle_send_exceptions and
backward_node_failure by adding more error regexes"

backward_node_failure by adding more error regexes**
backward_node_failure by adding more error regexes**
backward_node_failure by adding more error regexes**

backward_node_failure by adding more error regexes

Fixes these flaky tests.

Differential Revision: [D19584453](https://our.internmc.facebook.com/intern/diff/D19584453/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D19584453/)!

[ghstack-poisoned]

rohan-varma commented:
All failures are due to known flakiness.
